episodes/transfer_data.md

```diff
@@ -72,7 +132,81 @@ Write up your thoughts in our shared collaborative document.
 
 :::
 
-::::{dropdown} Push GitHub code
+## Share and manage code with GitHub
+
+GitHub is a popular platform for creating, storing, managing and sharing code with collaborators. For general guides for using the Git extensions for JupyterLab and RStudio, please refer to [here](https://blog.reviewnb.com/jupyterlab-git-extension/#git-workflow) and [here](https://docs.posit.co/ide/user/ide/guide/tools/version-control.html#integrations), respectively.
```

Comment:
I think this would benefit from another sentence, between the two already here, that explains what Git. (Can be brief, and limited to the context of GitHub and hub management.)

Reply:
```suggestion
GitHub is a popular platform for creating, storing, managing and sharing code with collaborators. This is useful for synchronising local code development on a hub with a remote GitHub repository. Known as version control, this system allows users to track changes to code over time and can facilitate collaborative work across different machines. For general guides for using the Git extensions for JupyterLab and RStudio, please refer to [here](https://blog.reviewnb.com/jupyterlab-git-extension/#git-workflow) and [here](https://docs.posit.co/ide/user/ide/guide/tools/version-control.html#integrations), respectively.
```

Reply:
Inspired by your start, I came up with this. What do you think?

```suggestion
GitHub is a popular platform for creating, storing, managing and sharing code with collaborators. Code developed on a hub can be synchronised with a remote repository on GitHub, and changes tracked in a process known as version control. This allows users to compare and even restore different versions of the code, and facilitates collaborative work across different machines. Version control and synchronisation of code with GitHub is enabled by a popular tool called Git. For general guides for using the Git extensions for JupyterLab and RStudio, please refer to [here](https://blog.reviewnb.com/jupyterlab-git-extension/#git-workflow) and [here](https://docs.posit.co/ide/user/ide/guide/tools/version-control.html#integrations), respectively.
```

Reply:
Finally, the link text in this paragraph is not very accessible: [we should avoid links with undescriptive text like "here", "this page", etc](https://webaim.org/techniques/hypertext/link_text#text).

Perhaps you can replace with something like: 

> On a hub, version control of code is facilitated by Git extensions to the development environment. Learn more about how to use these:
> - [Git extension for JupyterLab](https://blog.reviewnb.com/jupyterlab-git-extension/#git-workflow)
> - [Git extension for RStudio](https://docs.posit.co/ide/user/ide/guide/tools/version-control.html#integrations)

Reply:
```suggestion
GitHub is a popular *hosting platform* for creating, storing, managing and sharing code with collaborators. Code developed on a hub can be synchronised with a remote repository on GitHub, and changes are tracked in a process known as version control. This allows users to compare and even restore different versions of the code and facilitates collaborative work across different machines. Version control and synchronisation of code with GitHub is enabled by a popular tool called Git. On a hub, version control of code is facilitated by Git extensions to the development environment. Learn more about how to use these:

 - [Git extension for JupyterLab](https://blog.reviewnb.com/jupyterlab-git-extension/#git-workflow)
 - [Git extension for RStudio](https://docs.posit.co/ide/user/ide/guide/tools/version-control.html#integrations).
```

Reply:
Excellent 👍